pkg/kubectl/cmd/create.go

```diff
@@ -281,6 +281,9 @@ func NameFromCommandArgs(cmd *cobra.Command, args []string) (string, error) {
 	if len(args) == 0 {
 		return "", cmdutil.UsageErrorf(cmd, "NAME is required")
 	}
+	if len(args) > 1 {
```

Comment:
Maybe join the two `if`-s, like so:
```
if len(args) != 1 {
    return "", cmdutil.UsageErrorf(cmd, "exactly one NAME is required, got %d", len(args))
}
```
Also, I'm not a huge fan of warn-and-guess in general rather than failures.

Reply:
+1 invalid usage should just error out

Reply:
@ahmetb @apelisse Thanks for reviewing. I was thinking if have extra NAME, that's invalid usage, but the command can work. So no need to error out.
Error out also make sense. I will change accordingly. 
Thanks.

Reply:
But then it's more difficult to detect subtle bugs in scripts, and/or adds confusion for the user. It's better to just fail gracefully, and let the user know what happened and how to fix it.

Reply:
I see, thanks.
Can one of you help me ok-to-test, : (